File: tags/POST-3-ATTEMPT2/Projects/Python/Experiment/experiment.py
#! /usr/local/bin/python
import sys
sys.path = sys.path + ['./Build']
from RL_client_experiment import *

def run(num_episodes):
  for x in range(num_episodes):
    RL_episode(10)
    sys.stderr.write(".")


RL_init()
run(100)
RL_cleanup()

File: Projects/Network/Experiment/experiment.c
#include <stdio.h>
#include <RL_glue.h>

void run(int num_episodes)
{        
  int x = 0;
  for(x = 0; x < num_episodes; ++x)
  {
    RL_episode(10);
    fprintf(stderr, ".");
  }
}

int main(int argc, char *argv[])
{
  RL_init();
  run(100);
  RL_cleanup();

  return 0;
}
